airflow/providers/amazon/aws/operators/emr_create_job_flow.py

```diff
@@ -70,7 +71,7 @@ def execute(self, context):
         if isinstance(self.job_flow_overrides, str):
             self.job_flow_overrides = ast.literal_eval(self.job_flow_overrides)
 
-        response = emr.create_job_flow(self.job_flow_overrides)
+        response = emr.create_job_flow(self.job_flow_overrides)     # type: ignore
```

Comment:
You need to create a new variable. 
```
            job_flow_overrides = ast.literal_eval(self.job_flow_overrides)
        else:
            job_flow_overrides = self.job_flow_overrides
        response = emr.create_job_flow(job_flow_overrides)
```

Reply:
@mik-laj Thanks for the review. I just fixed it :D